service/gcs/runtime/runc/runc_test.go

```diff
@@ -34,9 +34,11 @@ func getBundlePath() (string, error) {
 	return filepath.Join(cwd, "testbundle"), nil
 }
 
+var createdContainers []runtime.Container
+
```

Comment:
Hmm, I see what you're saying. I was trying to mirror the old behavior (use the file system (global state) to find containers) to something similar (use global variables (global state) to find containers). But I'll see if I can eliminate this altogether.